privacy.md

```diff
@@ -0,0 +1,76 @@
+
+We have adopted the Australian Privacy Principles (APPs) contained in the Privacy Act 1988 (Cth) (the Privacy Act). The APPs govern the way in which we collect, use, disclose, store, secure and dispose of your Personal Information.
+
+A copy of the Australian Privacy Principles may be obtained from the website of The Office of the Australian Information Commissioner at <https://oaic.gov.au>.
+
+### What is Personal Information and why do we collect it?
+
+Personal Information is information or an opinion that identifies an individual. Examples of Personal Information we collect include: names, gender, addresses, email addresses, phone numbers, tertiary institutions, education information, and dietary requirements and/or medical conditions that you disclose to us in order to assist us help you throughout the event.
+
+This Personal Information is obtained in several ways including forms on our website <{{ site.url }}>, correspondence via email or phone calls, and the third parties listed below. We don’t guarantee website links or policy of authorised third parties.
+
+We collect your Personal Information for the primary purpose of providing our services to you and providing information to sponsors. You may unsubscribe from our mailing/marketing lists and opt-out from us providing information to our sponsors at any time by contacting us in writing.
+
+When we collect Personal Information we will, where appropriate and where possible, explain to you why we are collecting the information and how we plan to use it.
+
+### Third Parties
+
+We will primarily collect your Personal Information only from you. However, we also will be provided with information by several third parties, such as our third-part ticketing platform [Eventbrite](https://www.eventbrite.com.au/), payment provider [Pin Payments](https://pinpayments.com/), communications system [Slack](https://slack.com/) and hackathon management service, [DevPost](https://devpost.com/). In such a case we will take reasonable steps to ensure that you are made aware of the information provided to us by the third party.
+
+### Disclosure of Personal Information
+
+Your Personal Information may be disclosed in a number of circumstances including the following:
+
+* Third parties where you consent to the use or disclosure; and
+* Where required or authorised by law.
+
+### Security of Personal Information
+
+Your Personal Information is stored in a manner that reasonably protects it from misuse and loss and from unauthorized access, modification or disclosure.
+
+When your Personal Information is no longer needed for the purpose for which it was obtained, we will take reasonable steps to destroy or permanently de-identify your Personal Information. However, most of the Personal Information is or will be stored in participant files which will be kept by us for a minimum of 7 years.
```

Comment:
I think we might need to modify this statement a little as we do plan on keeping personal information for ongoing statistics, unless that is considered part of the reason we obtained it??

Reply:
@terencehuynh

Reply:
Is it C&S that keeps all event data for 7 years? If so, we don't really need things like address or medical conditions, but say gender or educational institution for diversity/school statistics we do use, so I agree that needs to be made clear.

Reply:
@sph-huynh, the '7 years' part comes from the Australian Government template I used, and I believe is related to [record keeping requirements](https://www.business.gov.au/info/plan-and-start/start-your-business/business-structure/change-business-structure/sole-trader-to-a-company/what-could-i-be-personally-liable-for-as-a-sole-trader-vs-a-company-director/what-records-do-i-need-to-keep) that corporations face (which I don't believe includes us).

Very happy to reduce that is C&S doesn't mandate a minimum.

Reply:
@dsykesturner ongoing statistics should be covered by this statement:

> we will take reasonable steps to destroy or permanently de-identify your Personal Information

since we can probably track all the statistics we need after de-identify the information?

Reply:
Okay sounds good :)